src/components/form-elements/_form-elements.scss

```diff
@@ -120,10 +117,12 @@ $include-html: false !default;
     &--full {
       width: 100%;
     }
+
     &--invalid {
       border: 2px solid $selectInvalidBorderColor;
+
       .mint-select__element {
-        padding: 0 $selectHeight/2+12px 0 ($selectHeight/2 - 2px);
+        padding: 0 $selectHeight/2 rhythm(0.5) 0 ($selectHeight/2 - rhythm(1/12));
```

Comment:
I agree maybe 0.5 looks better but we will use values like `2/3`probably so to be consistent lets use `1/2`